llpc/test/shaderdb/general/PipelineVsFs_TestPrimitiveID_First.pipe

```diff
@@ -0,0 +1,73 @@
+; Test primitiveID when provoking vertex mode is FIRST
+
+; BEGIN_SHADERTEST
+; RUN: amdllpc -spvgen-dir=%spvgendir% -v --gfxip=10.1.0 %s | FileCheck -check-prefix=SHADERTEST %s
+; SHADERTEST-LABEL: {{^// LLPC}} pipeline patching results
+; SHADERTEST: %{{[0-9]*}} = lshr i32 %{{[^,]*}}, 2
```

Comment:
Could you match the function signature of the parent function, so that we know where to look at when we want to manually inspect the output when this test stops passing in the future?

Reply:
Add more instructions. Thanks.